test/commands/new.test.ts

```diff
@@ -20,4 +20,12 @@ describe('new', () => {
       expect(ctx.stdout).to.equals('Created file specification.yaml...\n');
       done();
     });
+
+	test
+		.stdout()
+		.command(['new', '-h'])
```

Comment:
```suggestion
		.command(['new', '--help'])
```

try doing this

Reply:
Yes, I tried, getting the same error

Reply:
Yeah, not working only in the test environment 🤔

Reply:
found this though https://github.com/oclif/oclif/issues/142#issuecomment-406817611

Reply:
the reason is that actually `new --help` in test is not successful and throws:

```js
TypeError: cmd._help is not a function
```

I get the same when I try `bin/run new -h` with CLI. But `bin/run new --help` works fine when I try your branch 🤔

Reply:
@Souvikns maybe we should just resign from tests in this PR and write an issue about what I described above and get that solved first. Thoughts?

Reply:
ok, we need to go forward here:
@LakshyaSatpal please look at other comments, address them, and remove the test. There is no way to make it run now. The main functionality will have to stay untested 😄 

also please add `"additionalHelpFlags": ["-h"],` to oclif configuration in `package.json` (line 97) so we at least make `-h` to work too.

Reply:
Yeah, let's not add tests in this PR, but open an issue about how we cannot test the help messages. And once oclif fixes it we can add help tests for all other commands.